azure-pipelines.yml

```diff
@@ -28,6 +28,8 @@ steps:
     displayName: build ci
 
   - script: |
+      git config --global user.email "fabrictactical@microsoft.com"
```

Comment:
Should this be service.microsoft.com since it's a group? Or does plain microsoft.com also go to the group?

Reply:
This doesn't really matter, I'm just doing some git reads